proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -270,6 +270,9 @@ message WorkflowExecutionInfo {
     // (-- api-linter: core::0203::immutable=disabled
     //     aip.dev/not-precedent: field_behavior annotation is not yet used in this repo --)
     temporal.api.workflow.v1.WorkflowExecutionVersioningInfo versioning_info = 98;
+    // This filed is to record the transition history when the transition history is cleaned up due to disabling transition history
+    repeated VersionedTransition previous_transition_history = 99;
+    VersionedTransition previous_transition_history_break_point = 100;
```

Comment:
We need a list of break points I think. 
and should not have `previous` in the name because it will be used for when checking `transition_history`?

Reply:
```suggestion
    // This filed is to record the transition history when the transition history is cleaned up due to disabling transition history
    repeated VersionedTransition previous_transition_history = 99;
    VersionedTransition transition_history_break_point = 100;
```